### DISCUSSION TITLE
The shade of orange for the Github linguist color change

### DISCUSSION THREAD

**Discussion by Teddy55Codes** [ocaml/ocaml#12089] [Polls]
## What is this poll about?
This poll exists for the case that the [Github linguist color change to orange](https://github.com/ocaml/ocaml/discussions/12013) poll is accepted. If that happens this poll will decide the exact shade of orange that will be chosen for the Github color change. I know the difference between the 2 colors is small but i still want to make this poll to be sure just in case it is impotent to some people.

## `#ef7a08`
* The color originally proposed by [glennsl](https://discuss.ocaml.org/u/glennsl) on the [OCaml forum](https://discuss.ocaml.org/t/the-color-associated-with-ocaml-on-github-is-bad-lets-make-it-good/5597) in 2020.
* It has more contrast to html `#e34c26`.

## `#ee6a1a`
* The color that is used on the [OCaml website](https://ocaml.org/).

**Comment by glennsl** [comment]
Barely any difference between them, so I don't think it matters much, but here are links to see them better:

[#ef7a08](https://www.color-hex.com/color/ef7a08)
[#ee6a1a](https://www.color-hex.com/color/ee6a1a)

**Comment by Teddy55Codes** [comment]
The color change has been approved ([pull request](https://github.com/github-linguist/linguist/pull/6284)) and will be part of the next linguist release.
